impl/src/bitfield/config.rs

```diff
@@ -13,17 +13,17 @@ use syn::parse::Result;
 /// The configuration for the `#[bitfield]` macro.
 #[derive(Default)]
 pub struct Config {
-    pub bytes: Option<ConfigValue<usize>>,
-    pub bits: Option<ConfigValue<usize>>,
-    pub filled: Option<ConfigValue<bool>>,
-    pub repr: Option<ConfigValue<ReprKind>>,
-    pub derive_debug: Option<ConfigValue<()>>,
-    pub derive_specifier: Option<ConfigValue<()>>,
+    pub bytes: Option<Value<usize>>,
+    pub bits: Option<Value<usize>>,
+    pub filled: Option<Value<bool>>,
+    pub repr: Option<Value<ReprKind>>,
+    pub derive_debug: Option<Value<()>>,
+    pub derive_specifier: Option<Value<()>>,
```

Comment:
Those renamings are very subjective. I personally liked `ConfigValue` more since it is more on point.

Reply:
To be honest, me too, but clippy was complaining